hammer/hammer.go

```diff
@@ -208,22 +210,28 @@ func (h *Hammer) Run(ctx context.Context) {
 	}()
 }
 
-func newLeafGenerator(start uint64) func() []byte {
+func genLeaf(n uint64, minLeafSize int) []byte {
+	filler := make([]byte, minLeafSize/2)
```

Comment:
Can you pop a comment here that says we're using 1/2 of the size as filler because of the hex inflation?